kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py

```diff
@@ -0,0 +1,147 @@
+import asyncio
+import datetime
+import os
+from random import random
+
+import aiohttp
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import Context
+
+
+class EarthAndBeyondCog(commands.Cog, name='Earth and Beyond'):
+    _nasa_author_icon_url = 'https://i.imgur.com/7gnuJ1z.png'
+    _nasa_embed_url = 'https://i.imgur.com/Rac5kRM.png'
+    _color = 0xffffff
+    _error_display_text = "I'm very sorry, I'm looking up at the expanse of the universe just now. " \
+                          "I might be able to show another snapshot of the wonder a little later."
+
+    def __init__(self):
+        self._nasa_api_key = os.getenv('NASA_API_KEY', 'DEMO_KEY')
```

Comment:
I don't think we need the Demo key. I may have left it in the code by accident. My bad.

Reply:
That wasn't you, I added that for testing. I guess I forgot to remove it.

Reply:
Chances we both left it in there: Quite hight :D